near-sdk-macros/src/core_impl/abi/abi_embed.rs

```diff
@@ -2,7 +2,18 @@ use proc_macro2::TokenStream as TokenStream2;
 use quote::quote;
 
 pub fn embed() -> TokenStream2 {
-    let abi_path = env!("CARGO_NEAR_ABI_PATH");
+    let abi_path = match option_env!("CARGO_NEAR_ABI_PATH") {
+        Some(path) => path,
+        None => {
+            return quote! {
+                compile_error!(
+                    "the `__abi-embed` feature flag is private and should not be activated manually\n\
+                    \n\
+                    help\x1b[0m: consider installing https://github.com/near/cargo-near"
```

Comment:
```suggestion
                    help\x1b[0m: consider using https://github.com/near/cargo-near"
```

this wording may indicate that all one needs to do is install cargo-near and it will compile with `cargo check`, which isn't true. wdyt?

Reply:
lgtm.